src/mapper/utils.ts

```diff
@@ -117,24 +117,16 @@ export function getMappingAvailableSelectOptions(
           sheetId: sheetDefinition.id,
           sheetColumnId: column.id,
         },
+        used:
+          currentMapping.some(
+            (mapping) =>
+              mapping.sheetId === sheetDefinition.id &&
+              mapping.sheetColumnId === column.id
+          ) ?? false,
```

Comment:
when does `?? false` part is used? Can `some` function ever return null or undefined?